docs/docs/references/tunnel-cli/README.mdx

```diff
@@ -67,7 +68,43 @@ With the tunnel service in place, you can now develop and test your custom sign-
 
 ## Instructions
 
-### Step 1: Execute the command
+### Step 1: Install the CLI
+
+Install via your preferred package manager to use `logto-tunnel` command globally:
```

Comment:
with `npx` or `pnpx` no explicit installation is needed

Reply:
But the guide below also include a non-npx method to trigger the command. Maybe it's still better to mention installing it as a global dependency?

```
logto-tunnel ...
```